docs/advanced-entity-analytics/analyze-risk-score-data.asciidoc

```diff
@@ -0,0 +1,67 @@
+[[analyze-risk-score-data]]
+== View and analyze risk score data
+
+The {security-app} provides several options to monitor the change in the risk posture of hosts and users from your environment. We recommend that you prioritize alert triaging to identify anomalies or abnormal behavior patterns.
+
+Use the following places in the {security-app} to view and analyze risk score data:
```

Comment:
A jump list here would be helpful, to give readers a quick view of the places that the following sections describe.